tests/integrations/test_log_domain_kfac.py

```diff
@@ -76,7 +80,7 @@ def _train(nsteps, loss_and_grad, model_params, batch, key, logdomain=False):
             kernel = new_params["params"]["kernel"]
         model_params, optimizer_state = new_params, new_optimizer_state
 
-        training_results.append((kernel, bias, loss))
+        training_results.append((jnp.copy(kernel), jnp.copy(bias), jnp.copy(loss)))
```

Comment:
@ggoldsh btw, do you want to convert jax array to numpy array here? (see https://github.com/google/jax/issues/1961)

Reply:
Yeah I guess that's slightly cleaner, good idea